azurerm/internal/services/network/subnet_resource.go

```diff
@@ -97,19 +97,30 @@ func resourceArmSubnet() *schema.Resource {
 										Type:     schema.TypeString,
 										Required: true,
 										ValidateFunc: validation.StringInSlice([]string{
+											"Microsoft.ApiManagement/service",
+											"Microsoft.AzureCosmosDB/clusters",
 											"Microsoft.BareMetal/AzureVMware",
 											"Microsoft.BareMetal/CrayServers",
 											"Microsoft.Batch/batchAccounts",
 											"Microsoft.ContainerInstance/containerGroups",
 											"Microsoft.Databricks/workspaces",
+											"Microsoft.DBforMySQL/flexibleServers",
+											"Microsoft.DBforMySQL/serversv2",
+											"Microsoft.DBforPostgreSQL/flexibleServers",
 											"Microsoft.DBforPostgreSQL/serversv2",
+											"Microsoft.DBforPostgreSQL/singleServers",
 											"Microsoft.HardwareSecurityModules/dedicatedHSMs",
+											"Microsoft.Kusto/clusters",
 											"Microsoft.Logic/integrationServiceEnvironments",
+											"Microsoft.MachineLearningServices.workspaces",
```

Comment:
running the cli command this appears to be with a `/` matching the others, rather than a `.` - so can we make this:

```suggestion
											"Microsoft.MachineLearningServices/workspaces",
```

Reply:
oops, good catch.